sdks/python/build.gradle

```diff
@@ -226,6 +228,26 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
   }
 }
 
+task precommitIT(dependsOn: ['installGcpTest', 'sdist']) {
```

Comment:
This task should run in parallel to the rest of the precommit tasks. This can be done if it is placed in a separate sub-project. Sub-projects are created by creating a new build.gradle file in a subdirectory, such as `sdks/python/precommit/dataflow/build.gradle`. (example of making tests parallel: https://github.com/apache/beam/pull/5731/files)

Reply:
Leverage Gradle parallel mechanism is a great idea. I move some basic setup tasks into `BeamModulePlugins` and move `precommitIT` under `.../python/precommit/dataflow`. Let's see how this effects PreCommit build time.